app/test_main.py

```diff
@@ -1,3 +1,27 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, human_age",
+        [
+            pytest.param(-1, -1, [0, 0]),
+            pytest.param(0, 0, [0, 0]),
+            pytest.param(14, 14, [0, 0]),
+            pytest.param(15, 15, [1, 1]),
+            pytest.param(23, 23, [1, 1]),
+            pytest.param(24, 24, [2, 2]),
+            pytest.param(27, 27, [2, 2]),
+            pytest.param(28, 28, [3, 2]),
+            pytest.param(415, 510, [99, 99])
+        ]
+    )
+    def test_get_human_age(
+            self,
+            cat_age: int,
+            dog_age: int,
+            human_age: int
+    ) -> None:
+        assert get_human_age(cat_age, dog_age) == human_age
```

Comment:
![image](https://github.com/mate-academy/py-cat-and-dog-years/assets/93856423/82baa00a-32dd-403d-9133-63389fd1cfad)